docs/plugins/gitlabci-generic.md

```diff
@@ -0,0 +1,33 @@
+# `gitlabci-generic` Plugin
+
+This plugin creates Golang GitLab CI workflow.
+
+It downloads a template of your choice, render it with provided parameters, and creates a GitLab CI file to your repo.
+
+## Usage:
+
+_This plugin depends on an environment variable "GITLAB_TOKEN", which is your GitLab personal access token. Set it before using this plugin._
+
+See the [official document](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for more information.
```

Comment:
It would be nice to have a more specific description of what permissions the user needs at least